Packs/CommonTypes/IndicatorTypes/reputation-account.json

```diff
@@ -3,220 +3,223 @@
-              "args": {}
+        },
+        "displayname": {
+            "simple": "",
+            "complex": {
+                "root": "Account",
+                "filters": [],
+                "accessor": "DisplayName",
+                "transformers": [
+                    {
+                        "operator": "uniq",
+                        "args": {}
+                    }
+                ]
             }
-          ]
-        }
-      },
-      "domainname": {
-        "simple": "",
-        "complex": {
-          "root": "Account",
-          "filters": [],
-          "accessor": "Domain",
-          "transformers": [
-            {
-              "operator": "uniq",
-              "args": {}
+        },
+        "domainname": {
+            "simple": "",
+            "complex": {
+                "root": "Account",
+                "filters": [],
+                "accessor": "Domain",
+                "transformers": [
+                    {
+                        "operator": "uniq",
+                        "args": {}
+                    }
+                ]
             }
-          ]
-        }
-      },
-      "emailaddress": {
-        "simple": "",
-        "complex": {
-          "root": "Account",
-          "filters": [],
-          "accessor": "Email.Address",
-          "transformers": [
-            {
-              "operator": "uniq",
-              "args": {}
+        },
+        "emailaddress": {
+            "simple": "",
+            "complex": {
+                "root": "Account",
+                "filters": [],
+                "accessor": "Email.Address",
+                "transformers": [
+                    {
+                        "operator": "uniq",
+                        "args": {}
+                    }
+                ]
             }
-          ]
-        }
-      },
-      "groups": {
-        "simple": "",
-        "complex": {
-          "root": "Account",
-          "filters": [],
-          "accessor": "Groups",
-          "transformers": [
-            {
-              "operator": "uniq",
-              "args": {}
+        },
+        "groups": {
+            "simple": "",
+            "complex": {
+                "root": "Account",
+                "filters": [],
+                "accessor": "Groups",
+                "transformers": [
+                    {
+                        "operator": "uniq",
+                        "args": {}
+                    }
+                ]
             }
-          ]
-        }
-      },
-      "indicatoridentification": {
-        "simple": "",
-        "complex": {
-          "root": "Account",
-          "filters": [],
-          "accessor": "ID",
-          "transformers": [
-            {
-              "operator": "toUpperCase",
-              "args": {}
-            },
-            {
-              "operator": "uniq",
-              "args": {}
+        },
+        "indicatoridentification": {
+            "simple": "",
```

Comment:
indicatoridentification is not associated with that field

Reply:
Removed